### ISSUE TITLE
CUDA_PATH is set but CUDA wasnt able to be loaded. Please install the correct version of CUDA andcuDNN as mentioned in the GPU requirements page

### ISSUE THREAD

**Issue by zhoobin021** [cubiq/ComfyUI_InstantID#27]
please someone help me, I've been trying to solve this for hours but no luck.
I'm getting this error for everything, InstantID when using GPU, InstantID(zho), ReActor, ...basically anytime GPU(CUDA) is in use..

I've tried many things, 
-reinstalling nodes,
-Installing CUDA 12.2
-Installing CUDA 11.8
-Installing cudnn 8.9.2

but still getting this same error.
it works with CPU but I want to do it with GPU since my CPU is overheating and GPU is faster.

here's the error :
D:\a\_work\1\s\onnxruntime\python\onnxruntime_pybind_state.cc:857 onnxruntime::python::CreateExecutionProviderInstance CUDA_PATH is set but CUDA wasnt able to be loaded. Please install the correct version of CUDA andcuDNN as mentioned in the GPU requirements page (https://onnxruntime.ai/docs/execution-providers/CUDA-ExecutionProvider.html#requirements), make sure they're in the PATH, and that your GPU is supported.

**Comment by cubiq** [comment]
it means that your cuda version is not compatible with the onnxrunetime you have installed. Don't use cuda for insightface, it is not needed

**Comment by zhoobin021** [comment]
> it means that your cuda version is not compatible with the onnxrunetime you have installed. Don't use cuda for insightface, it is not needed

it worked, I installed CUDA 11.8 & cudnn 8.9.2 (but for CUDA11) and then I restarted my computer, and then everything worked.
I had to make it work because I also use FaceSwap, which only works with cuda I think since it gave me errors the same as InstantID with cuda. so basically fixed them both.
Thanks for the reply.

Here's a links for those having the same problem:

-CUDA Toolkit 11.8: 
https://developer.nvidia.com/cuda-11-8-0-download-archive
-cuddn 8.9.2 (for cuda 11.x) (June 1st, 2023):
https://developer.nvidia.com/rdp/cudnn-archive#:~:text=Download%20cuDNN%20v8.9.2%20(June%201st%2C%202023)%2C%20for%20CUDA%2011.x

Here's a tutorial on how to install them:
https://youtu.be/2TcnIzJ1RQs
